biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -2148,34 +2150,50 @@ subroutine LeafLayerMaintenanceRespiration_Atkin_etal_2017(lnc_top, &
-   real(r8), intent(out) :: lmr          ! Leaf Maintenance Respiration  (umol CO2/m**2/s)
+   real(r8), intent(in)  :: lnc_top          ! Leaf nitrogen content per unit area at canopy top [gN/m2]
+   integer,  intent(in)  :: ft               ! (plant) Functional Type Index
+   real(r8), intent(in)  :: vcmax25top       ! top of canopy vcmax
+   real(r8), intent(in)  :: cumulative_lai   ! cumulative lai above the current leaf layer
+   real(r8), intent(in)  :: veg_tempk        ! vegetation temperature  (degrees K)
+   real(r8), intent(in)  :: tgrowth          ! lagged vegetation temperature averaged over acclimation timescale (degrees K)
+   real(r8), intent(out) :: lmr              ! Leaf Maintenance Respiration  (umol CO2/m**2/s)
 
    ! Locals
    real(r8) :: lmr25   ! leaf layer: leaf maintenance respiration rate at 25C (umol CO2/m**2/s)
    real(r8) :: r_0     ! base respiration rate, PFT-dependent (umol CO2/m**2/s)
    real(r8) :: r_t_ref ! acclimated ref respiration rate (umol CO2/m**2/s)
    real(r8) :: lmr25top  ! canopy top leaf maint resp rate at 25C for this pft (umol CO2/m**2/s)
+   real(r8) :: rdark_scaler ! negative exponential scaling of rdark
+   real(r8) :: kn           ! decay coefficient
+   real(r8) :: rdark_decay  ! determines rate of rdark decay through canopy
 
+   
    ! parameter values of r_0 as listed in Atkin et al 2017: (umol CO2/m**2/s) 
    ! Broad-leaved trees  1.7560
    ! Needle-leaf trees   1.4995
    ! Shrubs              2.0749
    ! C3 herbs/grasses    2.1956
    ! In the absence of better information, we use the same value for C4 grasses as C3 grasses.
 
-   ! note that this code uses the relationship between leaf N and respiration from Atkin et al 
-   ! for the top of the canopy, but then assumes proportionality with N through the canopy.
-
+  
    ! r_0 currently put into the EDPftvarcon_inst%dev_arbitrary_pft
    ! all figs in Atkin et al 2017 stop at zero Celsius so we will assume acclimation is fixed below that
    r_0 = EDPftvarcon_inst%maintresp_leaf_atkin2017_baserate(ft)
-   r_t_ref = max( 0._r8, nscaler * (r_0 + lmr_r_1 * lnc_top + lmr_r_2 * max(0._r8, (tgrowth - tfrz) )) )
-   
+   rdark_decay = EDPftvarcon_inst%maintresp_leaf_decay(ft)
+
+   ! This code uses the relationship between leaf N and respiration from Atkin et al 
+   ! for the top of the canopy, but then scales through the canopy based on a rdark_scaler.
+   ! To assume proportionality with N through the canopy following Lloyd et al. 2010, use the
+   ! default parameter value of 2.43, which results in the scaling of photosynthesis and respiraiton
+   ! being proportional through the canopy. To have a steeper decrease in respiration than photosynthesis
+   ! this number can be smaller. There is some observational evidence for this being the case
+   ! in Lamour et al. 2023. 
+
+   kn = exp(0.00963_r8 * vcmax25top - rdark_decay)
```

Comment:
I know this is borrowed from Lloyd et al. (2010), but I wonder if the slope (0.00963) should be also made a parameter as they are both empirical.

Reply:
The kn equation is also used in other places in the photosynthesis subroutines. Would we want two sets of parameters - one for the photosynthesis scaling and one for respiration? Or just stick to respiration here and keep photosynthesis scaling parameters hard coded?

Reply:
I've pulled out the other parameter used in the rdark scaling. The other kn parameters are still hard coded.